This test timed out on powerpc-linux even after extending the
test_timeout_slow to 1800 seconds. Previously we tried to work around
this test by extending test_timeout_slow by 1.5 its previous value.

---
 gio/tests/meson.build | 5 ----
 1 file changed, 5 deletions(-)

diff --git a/gio/tests/meson.build b/gio/tests/meson.build
index 2631b1285..617f66238 100644
--- a/gio/tests/meson.build
+++ b/gio/tests/meson.build
@@ -459,11 +459,6 @@ if host_machine.system() != 'windows'
         'extra_sources' : [extra_sources, gdbus_test_codegen_generated, gdbus_test_codegen_generated_interface_info],
         'c_args' : ['-DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_32'],
       },
-      'gdbus-threading' : {
-        'extra_sources' : extra_sources,
-        'extra_programs': extra_programs,
-        'suite' : ['slow'],
-      },
       'gmenumodel' : {
         'extra_sources' : extra_sources,
         'suite' : ['slow'],
